src/journal/lookup3.c

```diff
@@ -967,29 +967,29 @@ void driver3()
   for (i=0, h=0; i<8; ++i)
   {
     h = hashlittle(buf, 0, h);
-    printf("%2ld  0-byte strings, hash is  %.8x\n", i, h);
+    printf("%2" PRIu32 "  0-byte strings, hash is  %.8x\n", i, h);
   }
 }
 
 void driver5()
 {
   uint32_t b,c;
   b=0, c=0, hashlittle2("", 0, &c, &b);
-  printf("hash is %.8lx %.8lx\n", c, b);   /* deadbeef deadbeef */
+  printf("hash is %" PRIx32 " %" PRIx32 "\n", c, b);   /* deadbeef deadbeef */
```

Comment:
the `.8` should stay I figure. here and everywhere

Reply:
fixed